config/betterquesting/DefaultQuests.json

```diff
@@ -307649,6 +307649,72 @@
           }
         }
       }
+    },
+    "2913:10": {
+      "questID:3": 2913,
+      "preRequisites:11": [
+        37
+      ],
+      "properties:10": {
+        "betterquesting:10": {
+          "snd_complete:8": "random.levelup",
+          "taskLogic:8": "AND",
+          "partySingleReward:1": 0,
+          "visibility:8": "NORMAL",
+          "isMain:1": 0,
+          "simultaneous:1": 0,
+          "icon:10": {
+            "id:8": "IC2:itemHarz",
+            "Count:3": 1,
+            "Damage:2": 0,
+            "OreDict:8": ""
+          },
+          "snd_update:8": "random.levelup",
+          "repeatTime:3": 18000,
+          "globalShare:1": 1,
+          "questLogic:8": "AND",
+          "repeat_relative:1": 1,
+          "name:8": "Sticky matter",
+          "isGlobal:1": 0,
+          "lockedProgress:1": 0,
+          "autoClaim:1": 0,
+          "isSilent:1": 0,
+          "desc:8": "Cant find any Rubber Tree? Can\u0027t build a water tank? Lazy to build a tree tap?  Have a stack of sticky resin for some coins."
```

Comment:
Cant find any Rubber Tree? Can\u0027t build a water tank? Lazy to build a tree tap? Have a stack of sticky resin for some coins.
should be
Can\u0027t find any Rubber Trees? Can\u0027t build a water tank? Too lazy to make a tree tap? Buy a stack of sticky resin for some coins.

This assumes the \u0027 means '

Reply:
'tank?Too' should have a space